projects/plugins/jetpack/modules/custom-css/csstidy/class.csstidy.php

```diff
@@ -279,6 +280,7 @@ function __construct() {
 		$this->settings['preserve_css'] = false;
 		$this->settings['timestamp'] = false;
 		$this->settings['template'] = ''; // say that propertie exist
+		$this->settings['preserve_css_variables'] = false;
```

Comment:
Just checking, this should be `true` during testing?

Reply:
I had toggle it on in `custom-css.php` but in the wrong place - should be fixed now, and have only enabled in block editor context for now